RickLeinecker/Fortuna
# Allow casus blocks to be dragged below the bottom casus block in the outer container

**Description**
Before Casus blocks couldn't be dragged outside the outermost container's bounding box. Now there is a special case added that fixed that.
Here is a picture for an example of what I mean:
![image](https://user-images.githubusercontent.com/18317476/76347753-81036800-62dd-11ea-8185-042222cbe4f2.png)

The console.log statement is added to the armory section because it is currently crashing. So that needs to be fixed by someone, but I think it is outside the scope of this PR.

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.

Casus still works as expected. Dragging a block below existing blocks now allows you to drop it there.

Passes flow.

**Final Checklist:**
Go down the list and check them off.
